composer.json

```diff
@@ -4,7 +4,8 @@
     "type": "library",
     "require": {
         "islandora/chullo": "^0.0.8",
-        "symfony/http-foundation": "3.2.6"
+        "symfony/http-foundation": "3.2.6",
+        "psr/log": "^1.0.1"
```

Comment:
I actually am using the monolog provider, but for the tests I wanted to use the generic test interfaces, so I needed that dependency.